src/store/messages/saga.ts

```diff
@@ -294,6 +294,7 @@ export function* stopSyncChannels(action) {
   );
 }
 
+// this also "recieves" the message which I had just "sent". is that ideal or should we handle it?
```

Comment:
Is this causing problems? i.e., are we double rendering or something? As long as we're properly just updating the right one then this is probably fine.

Reply:
i had just noticed it and put in a comment, currently it seems to behave properly.